table_processing/Table_processor_main.py

```diff
@@ -85,7 +87,12 @@ def console_main():
     logging.info("Table Processor started")
     input_file_path = input('Enter the path to the input PDF file: ')
     output_file_path = input('Enter the path to the output xlsx file: ')
-    output_file_path = process_pdf(input_file_path, output_file_path)
+    try:
+        input_intermediate_output = eval(input('Would you like to see the intermediate outputs? Answer True or False: ').lower())
```

Comment:
By setting this as lower case it makes it impossible to answer "True". I'll make a quick change to fix that.

Reply:
Issue with the prompt fixed